src/collections/Navigations/fields/topLevelNavTab.ts

```diff
@@ -1,27 +1,48 @@
-import { Tab, toWords } from 'payload'
+import { Field, Tab, toWords } from 'payload'
 import { itemsField } from './itemsField'
 
 export const topLevelNavTab = ({
   name,
   description,
-  isConfigurable = true,
+  hasConfigurableNavItems = true,
+  hasEnabledToggle = true,
 }: {
   name: string
   description?: string
-  isConfigurable?: boolean
-}): Tab => ({
-  name,
-  description,
-  virtual: !isConfigurable,
-  fields: [
+  hasConfigurableNavItems?: boolean
+  hasEnabledToggle?: boolean
+}): Tab => {
+  let fields: Field[] = [
     itemsField({
       label: `${toWords(name)} Nav Items`,
       description: `Dropdown items under ${toWords(name)}`,
       overrides: {
         admin: {
-          hidden: !isConfigurable,
+          hidden: !hasConfigurableNavItems,
         },
       },
     }),
-  ],
-})
+  ]
+
+  if (hasEnabledToggle) {
+    const enabledToggleField: Field = {
+      type: 'group',
+      name: 'options',
```

Comment:
```suggestion
      name: 'options',
      label: '',
```
Consider hiding the label since there is only one option

Reply:
Ahh yes! Forgot we could just pass an empty label.